os/include/tinyara/binfmt/binfmt.h

```diff
@@ -75,16 +75,16 @@
  * Public Types
  ****************************************************************************/
 enum {
-	ALLOC_TEXT,
+	BIN_TEXT,
 #ifdef CONFIG_BINFMT_CONSTRUCTORS
-	ALLOC_CTOR,
-	ALLOC_DTOR,
+	BIN_CTOR,
+	BIN_DTOR,
 #endif
-#ifdef CONFIG_OPTIMIZE_APP_RELOAD_TIME
-	ALLOC_RO,
-	ALLOC_DATA,
-#endif
-	ALLOC_MAX
+	BIN_RO,
```

Comment:
I think `BIN_RO` is used only when `CONFIG_OPTIMIZE_APP_RELOAD_TIME` is enabled.
Could you check this?